### PR TITLE
Add live page reloading feature #1326.

### PR DESCRIPTION
**Pull request by notetau**
This allows to reload automatically your browser when you modify contents.
Usage:
to start servers: "make devserver2_start PORT=8000 PORT2=9000"
to stop servers: "make devserver2_stop"
PORT is httpd port (you can see your page on http://localhost:PORT), PORT2 is websocket port to send a reload command to your browser.

### CODE REVIEW

**Comment by justinmayer** [comment]
Would appreciate some help from @getpelican/reviewers. Would you please have a look at this pull request and submit your review?

File: pelican/reload_server.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+
+        return True
+
+
+class ThreadedTCPServer(socketserver.ThreadingMixIn, socketserver.TCPServer):
+    pass
+socketserver.TCPServer.allow_reuse_address = True
+ThreadedTCPServer.daemon_threads = True
+
+PORT = len(sys.argv) == 2 and int(sys.argv[1]) or 9000
```
Comment (leotrs) [inline pelican/reload_server.py]: Consider `x if cond else y` syntax.

**Comment by leotrs** [comment]
@justinmayer Please let me know if this type of style-focused reviews seem useful to you.

**Comment by justinmayer** [comment]
@notetau: Many thanks for your contribution. In light of #2526, which has been merged, I think it's best if we close this pull request. If you are interested in building on top of the functionality added via that Invoke task, your participation would be most welcome.

**Pull request closed without merging**